R/expect_lint.R

```diff
@@ -158,3 +155,13 @@ expect_lint_free <- function(...) {
 
   invisible(result)
 }
+
+# Helper function to check if testthat is installed.
+require_testthat <- function(name) {
```

Comment:
I would not require the argument, e.g. `name = sys.call(sys.parent())`, or possibly re-using/unifying with `linter_auto_name()`. Then just `require_testthat()`.

Reply:
I tested this a bit `linter_auto_name(-5L)` seems to do the trick.